confluent_server/confluent/auth.py

```diff
@@ -115,6 +115,47 @@ class PromptsNeeded(Exception):
     def __init__(self, prompts):
         self.prompts = prompts
 
+ #add function to change _allowedbyrole and _deniedbyrole vars.
+ def add_roles(dictionary):
+    #function to parse the roles and the files. If there are modifications to be done to the roles, items will be added to dictionaries.
+    #If there are no moodifications done to one of the roles, it continues to the next
+    #Opening YAML file and reading the custom roles
+    with open("/etc/confluent/authorization.yaml","r") as stream:
+        loaded_file = yaml.safe_load(stream)
+        for outside_key,outside_value in loaded_file.items():
+            for inside_key,inside_value in outside_value.items():
+                try:
```

Comment:
You may want to look into the .update() method on python dictionaries to simplify the code